@kiva/kv-tokens/configs/kivaTypography.cjs

```diff
@@ -60,9 +61,10 @@ const textBaseColor = 'rgb(var(--text-primary))';
 */
 const textStyles = (() => {
 	const textJumbo = {
-		fontWeight: fontWeights.medium,
+		fontFamily: fonts.serif,
+		fontWeight: fontWeights.bold,
 		fontSize: rem(fontSizes.jumbo.sm),
-		letterSpacing: letterSpacings['-2'],
+		// letterSpacing: letterSpacings['-2'],
```

Comment:
Do we want to keep these commented out lines?

Reply:
Thanks for the reminder! I'll add to my list of items to bring up with design before deleting.